build-uclibc.sh

```diff
@@ -288,36 +289,45 @@ fi
 # make will fail if there is yet no .config file, but we can ignore this error.
 make distclean >> "${logfile}" 2>&1 || true 
 
-# Patch the temporary install directories used into the uClibc config.
-# uClibc 0.9.34 onwards use defconfig
-if [ ! -f extra/Configs/defconfigs/arc/defconfig ]
+# Copy the defconfig file to a temporary location
+TEMP_DEFCFG=`mktemp --tmpdir=${DEFCFG_DIR} XXXXXXXXXX_defconfig`
```

Comment:
That f***ing RHEL5 doesn't have mktemp that recognizes --tmpdir. We don't support rhel5 officially, but I use to build nightly builds, to ensure that toolchain will run on different farm machines... Need to think of something else.